opencraft/settings.py

```diff
@@ -209,7 +209,7 @@
 SWAMP_DRAGON_REDIS_PORT = REDIS_URL_OBJ.port
 SWAMP_DRAGON_CONNECTION = ('swampdragon.connections.sockjs_connection.DjangoSubscriberConnection', '/data')
 DRAGON_SERVER_ADDRESS_PORT = env('DRAGON_SERVER_ADDRESS_PORT', default='localhost:2001')
-DRAGON_URL = env('DRAGON_URL', default='http://{}/'.format(DRAGON_SERVER_ADDRESS_PORT))
+DRAGON_URL = env('DRAGON_URL', default='http://localhost:2001/')
```

Comment:
The current implementation uses the `DRAGON_SERVER_ADDRESS_PORT`, which is actually a bind address. So if I set that to `0.0.0.0:2001` to get it working with vagrant, the default `DRAGON_URL` becomes `http://0.0.0.0:2001/`, which is clearly wrong.